docs/src/markdown/ListItemDoc.mdx

```diff
@@ -1,7 +1,12 @@
+import { ImageGrid } from '../images/Image.tsx';
+import DrawerImg from './drawerAnatomy.png';
```

Comment:
We will likely have multiple "drawer" images. Can we be more descriptive and call this `DrawerAnatomy`?